packages/daisyui/src/daisyForm/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default } from './DaisyForm';
+export * from './DaisyForm';
```

Comment:
Adding blank line at end to remove the red circled dash
```suggestion
export * from './DaisyForm';

```